.github/workflows/test-runner-on-perf.yml

```diff
@@ -181,7 +190,7 @@ jobs:
             --enable-master-authorized-networks \
             --master-authorized-networks ${RESTORE_IPS}
       - name: "Notify Jade Slack"
-        if: always()
+        if:  ${{ github.event_name != 'workflow_dispatch' && always() }}
```

Comment:
Thoughts about notifying #jade-alerts all the time?

Reply:
Yeah, let's change that back to always()  This keeps the team in the loop that perf is currently getting worked on. If we find it's too noisy, we can change it back/move to another channel.